### DISCUSSION TITLE
Pre-Release tags

### DISCUSSION THREAD

**Discussion by kostas** [lovasoa/dezoomify-rs#241] [General]
Not important, but I would prefer to pull dezoomify-rs from homebrew. I noticed that the all three releases since [v2.11.2](https://github.com/lovasoa/dezoomify-rs/releases/tag/v2.11.2) have been tagged as "Pre-release", so they will not get pushed out to homebrew due to their policy of not accepting pre-realeases. Many of the changes do not seem very risky, so I am curious if marking these as "Pre-release" is intentional?

Are at some of these releases mature enough to promote to "Latest" tag?

**Comment by lovasoa** [comment]
Yes, this is intentional. Every single change to the code gets its own prerelease, but we don't want to make a new version every time.

**Comment by lovasoa** [comment]
Actually you are right, it looks like there is a bug in the release script: 

https://github.com/lovasoa/dezoomify-rs/blob/master/.github%2Fworkflows%2Foptimized-builds.yml#L53

**Comment by lovasoa** [comment]
Thanks for raising this. It should be fixed.

**Comment by kostas** [comment]
Thank you for the quick update. The change in the Latest tag already got [picked up by Homebrew](https://github.com/Homebrew/homebrew-core/commit/ab1a2d28a4f49a7314910ecaaed49e04acaa5072) and the v2.12.1 builds are already available!
